siriuspy/siriuspy/cycle/conn.py

```diff
@@ -619,11 +620,24 @@ def check_sofbmode(self, state, wait=1):
         state = _PSConst.OffOn.On if state == 'on' else _PSConst.OffOn.Off
         return _pv_timed_get(self['SOFBMode-Sts'], state, wait=wait)
 
+    def set_idffmode(self, state):
+        """Set IDFFMode."""
+        state = _PSConst.OffOn.On if state == 'on' else _PSConst.OffOn.Off
```

Comment:
maybe use state.lower() == 'on' just to also allow 'ON' and 'OFF'?

Reply:
done